ksql-functional-tests/src/test/resources/expected_topology/0_6_0-pre/key-field_-_stream___initially_set___partition_by_(same)___key_in_value___aliasing

```diff
@@ -51,20 +50,11 @@ Topologies:
       --> KSTREAM-MAPVALUES-0000000002
       <-- KSTREAM-SOURCE-0000000000
     Processor: KSTREAM-MAPVALUES-0000000002 (stores: [])
-      --> KSTREAM-FILTER-0000000003
+      --> KSTREAM-MAPVALUES-0000000003
```

Comment:
this topology has changed because the test was explicitly setting `ksql.query.fields.key.legacy` to `true`.  With this setting now gone the unnecessary repartition is avoided.